root/release/edit/tracklist.tt

```diff
@@ -24,6 +24,12 @@
           [%= l('Use track numbers') %]
         </label>
       </td>
+      <td>
+        <label>
+          [%= l('Custom time delimiter') %]
+          <input type="text" name="time-delimiter" data-bind="value: timeDelimiter" />
```

Comment:
It should be optional, that is, probably hidden behind an unchecked checkbox labelled with _Use custom delimiter_.